src/cs_transmog.cpp

```diff
@@ -199,25 +242,30 @@ class transmog_commandscript : public CommandScript
             return true;
         }
 
-        int locale = handler->GetSessionDbcLocale();
-        std::string setName = set->name[locale];
-        std::string nameLink = handler->playerLink(target->GetName());
+        bool isNotConsoleAndIsPlayerOnline = handler->GetSession() && target->GetSession();
```

Comment:
See issue for `isNotConsoleAndIsPlayerOnline` above.

Reply:
Not certain if relative but i had to use so.ething a bit different for the jailing command because of the crashing.
https://github.com/azerothcore/mod-anticheat/blob/master/src/cs_anticheat.cpp#L99

Maybe a simular technique needs to be deployed?

Reply:
Ah damn, I didn't see this in time.
I have a workaround now and added a PR for it. But it's not the prettiest. I'm gonna look into that later. Even if just out of curiosity :P

Reply:
It seems like the same issue though. It's always a connected player. So far I haven't found a way to get a player object from an offline player and also cannot create one because that requires a session to be initialized, which is stupid when the player isn't actually online.